doc/sphinx/proof-engine/detailed-tactic-examples.rst

```diff
@@ -829,21 +809,17 @@ Deciding intuitionistic propositional logic
             | |- (_ \/ _) => (left; TautoProp) || (right; TautoProp)
             end.
 
-The pattern matching on goals allows a complete and so a powerful
-backtracking when returning tactic values. An interesting application
```

Comment:
You removed this first sentence, but then we don't know "an interesting application" of what the first paragraph is talking about.